tests/embedding_lookup.cc

```diff
@@ -0,0 +1,38 @@
+#include <vector>
+#include <iostream>
+
+#include "embeddings.h"
+#include "finalfusion.h"
+
+int main() {
+  Embeddings *embeddings;
+  try {
+    embeddings = new Embeddings("data/test.fifu", false);
+  } catch (std::exception &e) {
+    return 1;
+  }
+
+  std::vector<float> embedding = embeddings->embedding("Berlin");
+  if (embedding.empty()) {
+    return 1;
```

Comment:
One of the reasons not to use bar pointers, this would like memory otherwise ;).